### PR TITLE
Remove comment for security

### PR DESCRIPTION
**Pull request by neocybereth**
Not sure if it's the case but I assume you may not want this in the code. Feel free to close if not

### CODE REVIEW

File: src/pages/feed.js [hunk 1]
```diff
@@ -82,62 +72,50 @@ export default class Feed extends Component {
...
-          style={{ maxHeight: '50vh', height: 'auto', width: 'auto' }}
-          src={`https://ipfs.io/ipfs/${
-            this.state.items[index].token_info.artifactUri.split('//')[1]
-          }`}
-          alt="💥"
-        />
-      </div>
-    )
+  setActiveTab = (tab) => {
+    console.log('yoo')
```
Comment (andrevenancio) [inline src/pages/feed.js]: 👋 Yo! but remove console log :)

Comment (neocybereth) [reply src/pages/feed.js]: :P will do!

Comment (neocybereth) [reply src/pages/feed.js]: Ah yeah looks like I accidentally pushed to master after submitting this, it's literally a one-liner. I'll fix that now

Comment (neocybereth) [reply src/pages/feed.js]: @andrevenancio now it's back in it's proper state :)

**Comment by andrevenancio** [comment]
Not at all. Deeply appreciate the PR. I think it's probably a good idea to write a contribution.md where we document the thinking moving forward. The idea is to try to update the website to the point it's as optimised and easy to maintain as possible. Hence my many comments, I was trying to communicate some of the thinking to move ahead and it's only fair that is written down to help anyone to contribute! I'll look into that!

**Pull request merged**